conan.cmake

```diff
@@ -586,25 +586,24 @@ macro(conan_config_install)
     endif()
 
     if(DEFINED CONAN_VERIFY_SSL)
-        set(CONAN_CONFIG_INSTALL_ARGS "${CONAN_CONFIG_INSTALL_ARGS} --verify-ssl ${CONAN_VERIFY_SSL}")
+        string(STRIP "${CONAN_CONFIG_INSTALL_ARGS} --verify-ssl=${CONAN_VERIFY_SSL}" CONAN_CONFIG_INSTALL_ARGS)
```

Comment:
This way of concatenating args, using a STRIP is a bit weird. I think I would have concat (using "=", that is correct), and then strip at the end if necessary. Am I missing something?

Reply:
Yes, it's a bit weird, in fact, I have just realized that there was another PR from an user solving this: https://github.com/conan-io/cmake-conan/pull/215 I'll close this one in favor of that one and add the test there.